app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+ 
+<div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, class: "input-field", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, class: "input-field", autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, class: "input-field", autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class: "input-field" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Kindly make your sign-in and sign-up pages match the design wireframe, you can move all your form elements to the center of the page 👍🏿 



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![blog_login (2)](https://github.com/faisalakb/R_blog_app/assets/69638013/488062aa-fefd-4eae-b6cf-83523260bc27) | ![Screenshot_244](https://github.com/firstpj/Ruby-blog/assets/69638013/3f8f88ec-d963-4172-bca4-cc5c2a972cab)

Reply:
Hi @firstpj did you align your form to the center of your page?
![Screenshot_245](https://github.com/firstpj/Ruby-blog/assets/69638013/d69f568e-9dec-4972-8fc9-95172b268673)